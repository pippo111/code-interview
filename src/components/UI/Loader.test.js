/* global describe, test, it, expect */

import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'

import { Loader } from '.'

describe('Loader', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<Loader />)
    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('renders custom caption', () => {
    const caption = 'Loading test...'

    const wrapper = mount(<Loader caption={caption} />)
    expect(wrapper.text()).toEqual(caption)
  })
})
