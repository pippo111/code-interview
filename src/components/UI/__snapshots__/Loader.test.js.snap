// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot renders 1`] = `
<div
  className="sc-AykKC hcXlKZ"
>
  Loading...
</div>
`;
